app/components/navigation_bar/view.html.erb

```diff
@@ -4,12 +4,15 @@
       <div class="moj-primary-navigation__nav">
         <nav class="moj-primary-navigation" aria-label="Primary navigation">
           <ul class="moj-primary-navigation__list">
+          <% items.each do |item| %>
             <li class="moj-primary-navigation__item">
-              <%= govuk_link_to "Home", home_path, { class: "moj-primary-navigation__link" } %>
-            </li>
-            <li class ="moj-primary-navigation__item">
-              <%= govuk_link_to "Trainee records", trainees_path, { class: "moj-primary-navigation__link" } %>
+              <% if item.fetch(:current, false) || current_page?(item.fetch(:url)) %>
```

Comment:
So this gives us the ability to pass in an item like `{ name: "Tab name", url: tab_path, current: true }` (with current = true),w and override it being rendered as active even though its url doesn't match the `current_url`.

You're not actually using this anywhere in the codebase though - the items are never supplied with `current: true`.